### PR TITLE
Cvars for command flood punishments

### PR DESCRIPTION
**Pull request by spark512**
Could we get something like this in the next build?

### CODE REVIEW

File: rehlds/rehlds/rehlds_security.cpp [hunk 1]
```diff
@@ -56,8 +61,13 @@ void CMoveCommandRateLimiter::CheckBurstRate(unsigned int clientId) {
 		dt = 0.2; //small intervals may give too high rates
 	}
 	if ((m_CurrentMoveCmds[clientId] / dt) > sv_rehlds_movecmdrate_max_burst.value) {
-		Cbuf_AddText(va("addip %i %s\n", 5, NET_BaseAdrToString(cl->netchan.remote_address)));
-		SV_DropClient(cl, false, "Banned for move commands flooding (burst)");
+		if(sv_rehlds_movecmdrate_burst_punish.value < 0) {
+			SV_DropClient(cl, false, "Kicked for move commands flooding (burst)");
+		}
+		else {
+			Cbuf_AddText(va("addip %i %s\n", sv_rehlds_movecmdrate_burst_punish.value, NET_BaseAdrToString(cl->netchan.remote_address)));
```
Comment (theAsmodai) [inline rehlds/rehlds/rehlds_security.cpp]: You must use %.1f here instead of %i.

**Comment by spark512** [comment]
You're right, updated.

**Comment by theAsmodai** [comment]
Thanks. And please add these cvars description to readme.md

**Comment by IgnacioFDM** [comment]
It would be nice to have a value (maybe -2) to "do nothing, just log"

**Pull request merged**